les/enr_entry.go

```diff
@@ -27,7 +27,8 @@ import (
 // lesEntry is the "les" ENR entry. This is set for LES servers only.
 type lesEntry struct {
 	// Ignore additional fields (for forward compatibility).
-	_ []rlp.RawValue `rlp:"tail"`
+	LesVersion, VfxVersion uint
```

Comment:
Can we remove the `LesVersion` now? It's not used. I don't think it's a good idea to add an unused field into the ENR.